docs/features/interactions.md

```diff
@@ -70,6 +70,8 @@ With the exception of render transforms (see the [pointer transform](https://git
 
 That said, you can simply throw away an old plot and replace it with a new one! This allows plotting of dynamic data: data which can change in real-time as it streams in, or because it is derived in response to external inputs such as range sliders and search boxes.
 
-On Observable, you can use [viewof](https://observablehq.com/@observablehq/views) in conjunction with [Observable Inputs](https://observablehq.com/@observablehq/inputs) for interactivity. If your cell references another cell, it will automatically re-run whenever the upstream cell’s value changes. For example, try dragging the slider in this [hexbin example](https://observablehq.com/@observablehq/plot-hexbin-binwidth?intent=fork). In React, use [useEffect](https://react.dev/reference/react/useEffect) and [useRef](https://react.dev/reference/react/useRef) to re-render the plot when data changes. In Vue, use [ref](https://vuejs.org/api/reactivity-core.html#ref). For more, see our [getting started guide](../getting-started.md).
+On Observable, you can use [viewof](https://observablehq.com/@observablehq/views) in conjunction with [Observable Inputs](https://observablehq.com/@observablehq/inputs) for interactivity. If your cell references another cell, it will automatically re-run whenever the upstream cell’s value changes. For example, try dragging the slider in this [hexbin example](https://observablehq.com/@observablehq/plot-hexbin-binwidth?intent=fork). The pointer transform (and by extension any automatic tip) will set the value property of the plot to the datum of the closest point, then fire an [*input* event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event). This allows to use a plot as an input.
```

Comment:
The purpose of this paragraph is to describe how to re-render plots in response to external interaction; it does not specify how the external interaction is implemented (other than referencing inputs, which is intended as an example). I think it would be better to explain the pointer transform’s support for viewof on the pointer transform documentation rather than here.